.github/workflows/greenplum-abi-tests.yml

```diff
@@ -93,7 +93,11 @@ jobs:
           sudo apt install -y libuv1-dev libpam0g-dev libldap-dev libipc-run-perl abi-dumper
           sudo ln -fs python2 /usr/bin/python
 
+        # FIXME: The next lines is only needed for the current pass of the ABI
+        # tests and should be removed after merging README.ubuntu.bash changes
+        # into the base branch.
       - name: Build libxerces 3.1.2
+        if: ${{ matrix.name == 'build-baseline' && matrix.ref == '6.27.1_arenadata58' }}
```

Comment:
Add to the PR description why we need this change

Reply:
Seems like in GitHub Actions, comparison operators like <= are not directly supported in conditional expressions (i.e., if conditions) for strings such as versions. See https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions.  About parsing strings: `Parsed from any legal JSON number format, otherwise NaN.` For more information, see the "[NaN Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN)."
So, let's save simple comparison `==`. It will be work fine in our case.

Reply:
Typos: "lebxerces", "wich". Write the branch name instead of "the base branch 6.x." for unambiguity

Reply:
fixed